src/main/cpp/chassis/swerve/driveStates/RobotDrive.cpp

```diff
@@ -35,6 +35,7 @@ RobotDrive::RobotDrive() : ISwerveDriveState::ISwerveDriveState(),
                            m_frState(),
                            m_blState(),
                            m_brState(),
+                           m_kinematics(ChassisFactory::GetChassisFactory()->GetSwerveChassis()->GetKinematics()),
                            m_wheelbase(units::length::inch_t(20.0)),
                            m_wheeltrack(units::length::inch_t(20.0)),
```

Comment:
do we need wheelbase, wheeltrack and center of rotatiion here anymore?

Reply:
I think we still need center of rotation to utilize that later.  However, wheel base and track can be removed